hibernate-core/src/main/java/org/hibernate/bytecode/enhance/internal/javassist/AttributeTypeDescriptor.java

```diff
@@ -79,6 +83,24 @@ public String buildInLineDirtyCheckingBodyFragment(JavassistEnhancementContext c
 		return builder.toString();
 	}
 
+	private void fetchValeBeforeComparison(JavassistEnhancementContext context, CtClass managedCtClass, CtField currentValue, StringBuilder builder) throws NotFoundException {
```

Comment:
"fetchValue...."?

Reply:
Done, method name was changed to : fetchValueBeforeComparisonIfNeeded
See last commit..

thanks.